src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/SNSConfiguration.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sns.SnsAsyncClient
+import java.net.URI
+
+@Configuration
+class SNSConfiguration(
+  @Value("\${aws.region}") private val region: String,
+  @Value("\${aws.access-key-id}") private val accessKeyId: String,
+  @Value("\${aws.secret-access-key}") private val secretAccessKey: String,
```

Comment:
_prefix_
I think we need specific access/secrets for the SNS integration. Is it possible to prefix them like that?

I.e. something like
```
aws.sns.region
aws.sns.access_key_id
aws.sns.secret_access_key
```

Reply:
sure thing